libs/platform/src/lib/components/menu/menu-trigger.directive.ts

```diff
@@ -103,9 +103,11 @@ export class MenuTriggerDirective implements OnDestroy, AfterContentInit {
     }
 
     /** @hidden Handled keypress which focus is on trigger element. */
+    @HostListener('spacebar', ['$event'])
     @HostListener('keydown', ['$event']) onTriggerKeydown($event: KeyboardEvent): void {
         switch ($event.key) {
             case 'Enter':
+            case ' ':
```

Comment:
is it not expected to also close the menu/select the item on spacebar? I see for `enter`, that on opening menu, if `enter` is pressed it selects the focused item; should this also happen for `spacebar` ?

Reply:
as per https://wiki.wdf.sap.corp/wiki/display/Acc/Menu, selection of menu item is only on Enter/Click not on spacebar.
but i see that Core and UI5 is also selecting menu-item on spacebar.
will check this with QA team.